README.md

```diff
@@ -11,6 +11,11 @@ It includes:
 * Quick run instructions
 * README automatic generation instructions
 
+
+> In order to make it possible to see your creations, you must assure that you have these pre-requisites.
+
+- [Roda Standalone](https://github.com/keeps/roda/tree/master/deploys) (Use Standalone Development, don't open WUI in spring boot)
```

Comment:
This link does not lead the developer to use docker-compose-dev, which causes a conflict with port 8080, please include the step-by-step guide here

Reply:
The command to generate the plugin documentation also does not work, please review.